lib/chef/provider/user/linux.rb

```diff
@@ -58,6 +58,7 @@ def universal_options
 
         def usermod_options
           opts = []
+          opts << "-u %d" % new_resource.uid if non_unique?
```

Comment:
Thanks, I'm new to ruby I basically copied things that were around. Would it be best to add another patch? Rebase?